### ISSUE TITLE
 Unhandled Exception: Unsupported operation: Cannot add to an unmodifiable list

### ISSUE THREAD

**Issue by yp19** [parse-community/Parse-SDK-Flutter#612]
I am following the ParseLiveListWidget example and getting the following error, I am using the **^3.0.0** version

```
E/flutter (11772): [ERROR:flutter/lib/ui/ui_dart_state.cc(186)] Unhandled Exception: Unsupported operation: Cannot add to an unmodifiable list
E/flutter (11772): #0      UnmodifiableListMixin.addAll (dart:_internal/list.dart:129:5)
E/flutter (11772): #1      ParseLiveList._runQuery (package:parse_server_sdk/src/utils/parse_live_list.dart:138:14)
E/flutter (11772): #2      ParseLiveList._init (package:parse_server_sdk/src/utils/parse_live_list.dart:154:47)
E/flutter (11772): #3      ParseLiveList.create (package:parse_server_sdk/src/utils/parse_live_list.dart:28:26)
E/flutter (11772): #4      new _ParseLiveListWidgetState (package:parse_server_sdk_flutter/src/utils/parse_live_list.dart:89:23)
E/flutter (11772): #5      ParseLiveListWidget.createState (package:parse_server_sdk_flutter/src/utils/parse_live_list.dart:51:49)
E/flutter (11772): #6      new StatefulElement (package:flutter/src/widgets/framework.dart:4584:24)
E/flutter (11772): #7      StatefulWidget.createElement (package:flutter/src/widgets/framework.dart:916:38)
E/flutter (11772): #8      Element.inflateWidget (package:flutter/src/widgets/framework.dart:3536:40)
E/flutter (11772): #9      Element.updateChild (package:flutter/src/widgets/framework.dart:3306:18)
E/flutter (11772): #10     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4520:16)
E/flutter (11772): #11     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:4667:11)
E/flutter (11772): #12     Element.rebuild (package:flutter/src/widgets/framework.dart:4189:5)
E/flutter (11772): #13     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:4474:5)
E/flutter (11772): #14     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:4658:11)
E/flutter (11772): #15     ComponentElement.mount (package:flutter/src/widgets/framework.dart:4469:5)
E/flutter (11772): #16     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3541:14)
E/flutter (11772): #17     Element.updateChild (package:flutter/src/widgets/framework.dart:3306:18)
E/flutter (11772): #18     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4520:16)
E/flutter (11772): #19     Element.rebuild (package:flutter/src/widgets/framework.dart:4189:5)
E/flutter (11772): #20     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:4474:5)
E/flutter (11772): #21     ComponentElement.mount (package:flutter/src/widgets/framework.dart:4469:5)
E/flutter (11772): #22     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3541:14)
E/flutter (11772): #23     Element.updateChild (package:flutter/src/widgets/framework.dart:3306:18)
E/flutter (11772): #24     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4520:16)
E/flutter (11772): #25     Element.rebuild (package:flutter/src/widgets/framework.dart:4189:5)
E/flutter (11772): #26     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:4474:5)
E/flutter (11772): #27     ComponentElement.mount (package:flutter/src/widgets/framework.dart:4469:5)
E/flutter (11772): #28     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3541:14)
E/flutter (11772): #29     Element.updateChild (package:flutter/src/widgets/framework.dart:3306:18)
E/flutter (11772): #30     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4520:16)
E/flutter (11772): #31     Element.rebuild (package:flutter/src/widgets/framework.dart:4189:5)
E/flutter (11772): #32     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:4474:5)
E/flutter (11772): #33     ComponentElement.mount (package:flutter/src/widgets/framework.dart:4469:5)
E/flutter (11772): #34     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3541:14)
E/flutter (11772): #35     MultiChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:6094:32)
E/flutter (11772): #36     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3541:14)
E/flutter (11772): #37     Element.updateChild (package:flutter/src/widgets/framework.dart:3306:18)
E/flutter (11772): #38     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4520:16)
E/flutter (11772): #39     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:4667:11)
E/flutter (11772): #40     Element.rebuild (package:flutter/src/widgets/framework.dart:4189:5)
E/flutter (11772): #41     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:4474:5)
E/flutter (11772): #42     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:4658:11)
E/flutter (11772): #43     ComponentElement.mount (package:flutter/src/widgets/framework.dart:4469:5)
E/flutter (11772): #44     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3541:14)
E/flutter (11772): #45     Element.updateChild (package:fl
```

**Comment by yp19** [comment]
Any work around?? I tried to both Stateful and Stateless widgets.

**Comment by yp19** [comment]
@RodrigoSMarques Could you help with this. Thanks

**Comment by fischerscode** [comment]
@yp19 
This should get fixed with  #640.

**Comment by RodrigoSMarques** [comment]
@yp19 
New version 3.1.0 available. Check if the error persists.
